### PR TITLE
Use github issues instead of redmine

### PR DESCRIPTION
**Pull request by fao89**
Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

**Review by dralley — APPROVED** [review]

File: template_config.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ test_fips_nightly: false
 test_performance: false
 test_released_plugin_with_next_pulpcore_release: false
 test_s3: true
-update_redmine: true
+update_redmine: false
```
Comment (ggainey) [inline template_config.yml:76]: Is there a reason to keep this stanza at all any more? Is there something we should be doing in an "update_github: true" stanza instead?

**Review by bmbouter — APPROVED** [review]
Looks good to me. Thank you!

File: template_config.yml (hunk 1 above)
Comment (bmbouter) [reply template_config.yml:76]: Yeah we should be removing this from the plugin template +1

Comment (fao89) [reply template_config.yml:76]: yep, I'll start a PR on plugin-template to remove or update it

**Pull request merged**